src/main/java/org/skriptlang/skript/bukkit/spawner/elements/expressions/spawnerentry/equipment/ExprEquipmentWithDropChances.java

```diff
@@ -0,0 +1,97 @@
+package org.skriptlang.skript.bukkit.spawner.elements.expressions.spawnerentry.equipment;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.EquipmentSlot;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.spawner.SpawnerModule;
+import org.skriptlang.skript.bukkit.spawner.util.SpawnerEntryEquipment;
+import org.skriptlang.skript.bukkit.spawner.util.SpawnerEntryEquipment.Drops;
+import org.skriptlang.skript.registration.SyntaxInfo;
+import org.skriptlang.skript.registration.SyntaxOrigin;
+import org.skriptlang.skript.registration.SyntaxRegistry;
+
+import java.util.ArrayList;
+
+@Name("Spawner Entry - Equipment with Drops")
+@Description("Returns the drops of a spawner entry equipment.")
+@Examples("set {_chances::*} to drop chances of {_equipment}")
+@Since("INSERT VERSION")
+@RequiredPlugins("Minecraft 1.21+")
+public class ExprEquipmentWithDropChances extends PropertyExpression<SpawnerEntryEquipment, Drops> {
+
+	static {
+		var info = SyntaxInfo.Expression.builder(ExprEquipmentWithDropChances.class, Drops.class)
+			.origin(SyntaxOrigin.of(Skript.instance()))
+			.supplier(ExprEquipmentWithDropChances::new)
+			.priority(PropertyExpression.DEFAULT_PRIORITY)
+			.addPatterns(
+				"[the] drop chance[s] (from|of) %spawnerentryequipments%",
```

Comment:
same as previous, add `spawner` or something to make it more precise to what its doing. Spawners arent the only thing on the server that have a drop chance.